lib/matplotlib/collections.py

```diff
@@ -1171,29 +1171,52 @@ def __init__(self, segments,     # Can be None.
                  **kwargs
                  ):
         """
-        *segments*
-            a sequence of (*line0*, *line1*, *line2*), where::
+        Parameters
+        ----------
+        segments :
+            A sequence of (*line0*, *line1*, *line2*), where::
 
                 linen = (x0, y0), (x1, y1), ... (xm, ym)
 
             or the equivalent numpy array with two columns. Each line
             can be a different length.
 
-        *colors*
-            must be a sequence of RGBA tuples (e.g., arbitrary color
+        colors : sequence, optional
+            A sequence of RGBA tuples (e.g., arbitrary color
             strings, etc, not allowed).
 
-        *antialiaseds*
-            must be a sequence of ones or zeros
+        antialiaseds : sequence, optional
+            A sequence of ones or zeros.
```

Comment:
*Does someone knows what this option actually does?* At least enough to suggest a sentence giving a rough idea of its effect or purpose ^^.

Reply:
it controls if any given line segment is anti-aliased when drawn.  This gets handed through to the `gc` object and may or may not be respected by all of the backends.  I think it should be read as (antialiased (s) like linestyle (s)).